src/client/App.tsx

```diff
@@ -41,7 +41,7 @@ const App = () => {
         </div>
       </div>
       <div className="app__footer">
-        <p>Footer </p>
+        <p></p>
```

Comment:
we should not keep any paragraph without content. If the footer css rules are applied to the `p` element, then move them accordingly to his parent `<div className="app__footer">`